helix-term/src/ui/completion.rs

```diff
@@ -92,8 +92,10 @@ impl menu::Item for CompletionItem {
                                 value, ..
                             }) => value,
                         };
-                        Color::from_hex(text)
+                        // LSPs which send Color completion items include a 6 digit hex code at the end for the color. The extra 1 digit is for the '#'
```

Comment:
```suggestion
                        // Language servers which send Color completion items tend to include a 6
                        // digit hex code at the end for the color. The extra 1 digit is for the '#'
```

minor wording nit about "LSPs" vs "language servers", but also let's say that they "tend to" send the hex code in this format since it's a convention rather than a part of the LSP spec

Reply:
yeah, this makes sense